src/renderers/noop/ReactNoopEntry.js

```diff
@@ -430,10 +430,10 @@ var ReactNoop = {
       if (fiber.updateQueue) {
         logUpdateQueue(fiber.updateQueue, depth);
       }
-      const childInProgress = fiber.progressedChild;
+      const childInProgress = fiber.child;
       if (childInProgress && childInProgress !== fiber.child) {
```

Comment:
This branch is now always dead. Might be better to just comment it out so we know what it is supposed to do.